SMLHelper/Patchers/OptionsPanelPatcher.cs

```diff
@@ -21,7 +21,21 @@ internal static void Patch(HarmonyInstance harmony)
         internal static void AddTabs_Postfix(uGUI_OptionsPanel __instance)
         {
             var optionsPanel = __instance;
-            var modsTab = optionsPanel.AddTab("Mods");
+            
+            var modsTab = -1;
+            for (int i = 0; i < optionsPanel.tabsContainer.childCount; i++)
```

Comment:
This all looks good, but is also going to be something that we will forget about later.  
Wrap this up into a method with a descriptive name or add enough comments to inform why this is being done.

Reply:
Addressed in commit 586ddb5